lib/sen_tweet_web/router.ex

```diff
@@ -27,7 +27,7 @@ defmodule SenTweetWeb.Router do
 
   scope "/admin", SenTweetWeb do
     pipe_through([:browser, :auth])
-
+    get("/streams", PageController, :stream_admin)
```

Comment:
I recommend using [resources](https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4) here rather than `get` since we'll have multiple actions (GET, POST, DELETE) for streams.

The resource is the stream so it makes sense to create a `StreamController`.

Reply:
OK, i set up a stream controller:
```elixir
defmodule SenTweetWeb.StreamController do
  use SenTweetWeb, :controller

  def index(conn, _params) do
      json(conn, %{streams: Bitfeels.all_streams()})
  end
end
```
and modified the `/admin` scope:
```elixir
scope "/admin", SenTweetWeb do
    resources "/streams", StreamController
    pipe_through([:browser, :auth])
    live_dashboard("/dashboard", metrics: SenTweetWeb.Telemetry)
end
```
![image](https://user-images.githubusercontent.com/31669884/84445174-60ff7f80-ac11-11ea-90b6-3b5c53deee42.png)

I'm thinking the `StreamController` should just handle the requests to GET, PUT, DELETE streams and the `PageController` should actually render the stream-admin page. Am i thinking about this right?

Reply:
Not quite, we don't need to be rendering any JSON. We want a collection of pages that we can list, create, and delete streams. All this functionality is part of the `StreamController` scope.

The `PageController` is just for the homepage and is not related to anything you are doing for this PR.

Reply:
I recommend running `mix phx.routes` and look at what routes resources generated for the stream controller. You'll want to follow that pattern and _fill in the blanks_.

Reply:
Ok, this is more clear now. I guess I was confused by the name `PageController` as it implied to me it should handle rendering all pages, so I was thinking `StreamController` should act like API.

Reply:
Yeah it's confusing. `PageController` is the generic name phoenix gave that controller when you first generate a phoenix application. The name is completely meaningless 🤷‍♂️ 

Just looking for an HTML page that lists the streams but under the `index` action on the `StreamController`

Reply:
yep, I have that working now. just prettying things up and adding the buttons :fire:

Looking ahead, I'll need a unique ID for each stream in order to implement the `delete` route. I could make this the index of the list that `Bitfeels.all_streams()` returns, but that seems hacky.  Maybe I could make it a string "user-track" and use pattern matching?